packages/components/src/box-control/input-controls.tsx

```diff
@@ -49,7 +55,9 @@ export default function BoxInputControls( {
 			 * Supports changing pair sides. For example, holding the ALT key
 			 * when changing the TOP will also update BOTTOM.
 			 */
-			if ( altKey ) {
+			// @ts-expect-error - event.altKey is only present when the change event was
+			// triggered by a keyboard event.
+			if ( event.altKey ) {
```

Comment:
I'll add a TODO comment here because I think it should be addressed in a different layer, for example in how the upstream change handler passes events. c64c2a61